test/include/dlaf_test/matrix/util_generic_blas.h

```diff
@@ -61,22 +63,22 @@ auto getSubMatrixMatrixMultiplication(const SizeType a, const SizeType b, const
 
   auto elC = [a, b, k](const GlobalElementIndex ij) {
     if (ij.row() < a || ij.row() > b || ij.col() < a || ij.col() > b)
-      return TypeUtilities<T>::polar(-99, 99);
+      return -TypeUtilities<T>::polar(99, 99);
 
     const double i = ij.row();
     const double j = ij.col();
 
     return TypeUtilities<T>::polar((i + k + 1) / (j + 5), i + j + k);
   };
 
-  auto elR = [a, b, k, alpha, beta](const GlobalElementIndex ij) {
+  auto elR = [a, b, sub, k, alpha, beta](const GlobalElementIndex ij) {
     if (ij.row() < a || ij.row() > b || ij.col() < a || ij.col() > b)
-      return TypeUtilities<T>::polar(13, -26);
+      return -TypeUtilities<T>::polar(99, 99);
```

Comment:
Were `13` and `-26` "important numbers" or is `99, 99` just as good (or better) in this situation? Asking because I can't judge if it's a copy-paste mistake or intentional.

Reply:
It is the bug-fix I'm talking about in the first point of the changelog 😉